src/main/java/persistence/sql/ddl/QueryBuilder.java

```diff
@@ -0,0 +1,76 @@
+package persistence.sql.ddl;
+
+import jakarta.persistence.*;
+import org.jetbrains.annotations.NotNull;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class QueryBuilder {
+    Class<?> clazz;
+    SqlTypeMapper sqlTypeMapper;
+
+    public QueryBuilder(Class<?> clazz, SqlTypeMapper sqlTypeMapper) {
+        this.clazz = clazz;
+        this.sqlTypeMapper = sqlTypeMapper;
+    }
+
+    public String create() {
+        return "CREATE TABLE " + getTableName() + "(" + getPersonMetadata() + ")";
+    }
+
+    public String drop() {
+        return "DROP TABLE " + getTableName() + " IF EXISTS";
+    }
+
+    private String getTableName() {
+        if (clazz.isAnnotationPresent(Table.class)) {
+            return clazz.getAnnotation(Table.class).name();
+        }
+        return clazz.getSimpleName();
+    }
+
+    public String getPersonMetadata() {
```

Comment:
1. 객체 내부에서만 사용하는 메서드들은 접근제어자를 private으로 설정해주시면 좋습니다! 👍 

2. 메서드 이름이 특정 클래스에 종속되면 코드의 유연성과 재사용성이 떨어지기 때문에 메서드가 하는 일을 잘 표현할 수 있도록 바꿔주시는 것이 적절할 것 같습니다!

```suggestion
    private String getColumnQueries() {
```